pkg/steps/template.go

```diff
@@ -50,6 +53,22 @@ type templateExecutionStep struct {
 	subTests []*junit.TestCase
 }
 
+func injectResourcesToPod(pod *coreapi.Pod, templateName string, resources api.ResourceConfiguration) error {
+	containerResources, err := resourcesFor(resources.RequirementsForStep(templateName))
```

Comment:
That will return the resources for the specific template, but if there is nothing defined it will return the '*' resources.